builders/interoperability/xcm/xc20/.pages

```diff
@@ -3,6 +3,5 @@ nav:
   - index.md
   - 'Overview': 'overview.md'
   - 'External XC-20s': 'xc20.md'
-  - 'Mintable XC-20s': 'mintable-xc20.md'
-  - 'Send XC-20s': 'xtokens.md'
-
+  - 'Send External XC-20s': 'xtokens.md'
+  - 'Send Local ERC-20s via XCM': 'xcm-erc20s.md'
```

Comment:
moved this to the bottom as it doesn't follow the same format as the 'External XC-20s' page and more aligns with the 'Send External XC-20s' page. Not sure about the name, but I did want to make it clear for those searching that this page is related to sending ERC-20s (for SEO purposes)